src/ObjectInfo/DoctrineORMInfo.php

```diff
@@ -62,19 +62,19 @@ private function getAssocsConfig(ClassMetadata $metadata, array $assocNames): ar
         $assocsConfigs = [];
 
         foreach ($assocNames as $assocName) {
-            if (!$metadata->isAssociationInverseSide($assocName)) {
+            $associationMapping = $metadata->getAssociationMapping($assocName);
+
+            if (isset($associationMapping['inversedBy'])) {
                 continue;
             }
 
             $class = $metadata->getAssociationTargetClass($assocName);
 
             if ($metadata->isSingleValuedAssociation($assocName)) {
-                $nullable = ($metadata instanceof ClassMetadataInfo) && isset($metadata->discriminatorColumn['nullable']) && $metadata->discriminatorColumn['nullable'];
-
                 $assocsConfigs[$assocName] = [
                     'field_type' => AutoFormType::class,
                     'data_class' => $class,
-                    'required' => !$nullable,
+                    'required' => false,
```

Comment:
If I don't change this, association fields are always created empty. Shouldn't this be just required false directly?